CMakeLists.txt

```diff
@@ -64,10 +64,14 @@ macro(build_console_bridge)
 
   include(ExternalProject)
 
-  # Get console_bridge 0.5.1
-  externalproject_add(console_bridge-1.0.1
-    URL https://github.com/ros/console_bridge/archive/1.0.1.tar.gz
-    URL_MD5 9f5351994c7690b37f88e9f70e478061
+  # Download and build console_bridge
+  externalproject_add(console_bridge-1.0.2
+    GIT_REPOSITORY https://github.com/ros/console_bridge.git
+    GIT_TAG 0828d846f2d4940b4e2b5075c6c724991d0cd308 # 1.0.2
+    GIT_CONFIG advice.detachedHead=false
+    # Suppress git update due to https://gitlab.kitware.com/cmake/cmake/-/issues/16419
+    # See https://github.com/ament/uncrustify_vendor/pull/22 for details
+    UPDATE_COMMAND ""
```

Comment:
Now that we have a tagged upstream version, and don't need to patch anymore, can we switch back to `URL`/`URL_MD5`?

Reply:
Back to using `URL` and a hash in b654310